bitnami/thanos/templates/querier/ingress.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
-  name: {{ include "thanos.fullname" . }}
+  name: {{ include "thanos.fullname" . }}-querier
```

Comment:
This is a breaking change (imho) for tools that connect to that ingress so I think we either need to bump the minor version or mention this in the README file.

Reply:
Also is there a real need to rename the ingress?

Reply:
No pain - no gain. There's an Upgrade note in the `README` now and the chart was bumped to `2.4.0`. 🌞

Reply:
Hi,

Could you elaborate why renaming the ingress would break other tools? I would like to know an example.

Reply:
@javsalgar no particular example in mind. I was mostly thinking external tooling that treats the object an as API. For example, think of a script that fetches the ingress object and extracts some info from the json like
`kubectl get ing thanos -o json | jq ....` . I am not sure really if we should be treating the objects themselves as part of some API that needs to follow the same semver rules.

Reply:
Yeah - no obvious issues with the name change of the resource.

Even so it's still a change of the name of a resource. So should we keep the minor version bumped? I think so.

Reply:
In my opinion, I think that the major change should focus on the chart itself. I'm not sure until what extent we should guarantee that no tool around would break. I also believe it should be a minor change in this case. However, thanks for the input, I will forward it to the rest of the team for future retrospective